### ISSUE TITLE
Search doesn't return expected results—01

### ISSUE THREAD

**Issue by sabbamitta** [sc-voice/sc-voice#258]
#### Bug Description
scv-bilara doesn't return results for "At one time the Buddha was staying near Benares, in the deer park at Isipatana"; setting: de-search -ml 3; branch: sn56.11_de_sabbamitta; there are already two suttas with this exact term translated, but they are not shown. Only the instance in sn56.11 is shown, but since this is the sutta in progress, the "de" line has just a copy of the "en" line.

For the same search in the master branch I get the correct results, but only if I add "-sl en"; otherwise it assumes the term is somewhat between Pali and German:
```
pattern      : "at one time the buddha was staying near benares, in the deer park at isipatana" grep:\bat|\bone|\btime|\b(t|ṭ)he|\bb(u|ū)(d|ḍ)(d|ḍ)h(a|ā)|\bw(a|ā)s|\bstaying|\bnear|\bbenares,|\b(i|ī)(n|ṅ|ñ|ṇ)|\b(t|ṭ)he|\bdeer|\bpark|\bat|\b(i|ī)s(i|ī)p(a|ā)(t|ṭ)(a|ā)(n|ṅ|ñ|ṇ)(a|ā)
languages    : translation:de search:de minLang:3
output       : human color:201 elapsed:0.316s maxDoc:50
found        : segs:0 by:keywords mlDocs:0 docs:0
```

> Today I was searching for the phrase “At one time the Buddha was staying near Benares, in the deer park at Isipatana”, which is the first sentence in the Dhammacakkappavattana Sutta. I was absolutely sure I had already translated that sentence and just entered it as a whole—but the only result I got was the sutta I am currently translating (the Dhammacakka, sn56.11), with that phrase both for English and German. When scv-bilara searches in the translation branch, the sutta-in-progress is of course already there, but just with a replication of the English for the German line.
> 
> Well, so I narrowed down my search term, thinking maybe that other instance was a little bit different. Whatever I tried, no other result. Until I entered just the word “benares”: Then I got three results (one of them SN56.11) which means that my term is already in two other suttas. And it is exactly the same sentence, not even the slightest comma is different!

#### Discovered on:
 - OS: [e.g. Ubuntu 16.4]
 - Browser & Version: [e.g. chrome, safari]

**Comment by sabbamitta** [comment]
merging with #257 because it is an accidental duplication.

**Comment by firepick1** [comment]
Oddly, this problem was indeed a word special case. It thought benares, bhaddiya, kosambi to be Deutsch. I have since trained scv-bilara to know them as English. 

Fixed in v1.2.8 scv-bilara

**Comment by sabbamitta** [comment]
> to know them as English

Why English and not Pali? They will also be in German now… :thinking:
